journal/modernized/13-chapter-12.adoc

```diff
@@ -78,9 +78,6 @@ as this is obeyed and followed we shall experience our spirits refreshed and
 strengthened by the life-giving presence of Him who is the minister of the
 sanctuary and true tabernacle which God has pitched,
 and not man.
-you not be pleased to succor me, and help me;
-blot out my transgressions from your book of remembrance for Jesus Christ`'s sake.
-Amen, and amen.
```

Comment:
again here... the chunck deleted (from p. 323) is already on the third paragraph (same page). Also checked with the original book from google.